linux/net/rina/serdes.c

```diff
@@ -1065,3 +1101,82 @@ struct pdu * pdu_deserialize_ni(const struct serdes * instance,
                                 struct crypto_blkcipher * blkcipher)
 { return pdu_deserialize_gfp(GFP_ATOMIC, instance, pdu, dup_conf, blkcipher); }
 EXPORT_SYMBOL(pdu_deserialize_ni);
+
+int serdes_pci_size(pdu_type_t type, struct dt_cons * c) {
```

Comment:
My point is that in the code as it is now, it is not needed, it is only used by ```serdes_pdu_size```. If we eventually need to access it from somewhere else, it can then be exported.